tla/consistency/TraceMultiNodeReads.tla

```diff
@@ -0,0 +1,137 @@
+-------------------------------- MODULE TraceMultiNodeReads -------------------------------
+EXTENDS MultiNodeReads, Json, IOUtils, Sequences
+
+\* Trace validation has been designed for TLC running in default model-checking
+\* mode, i.e., breadth-first search.
+ASSUME TLCGet("config").mode = "bfs"
```

Comment:
Perhaps move/combine with where the spec assumes a single worker.

Reply:
Sure, same as above though.